src/tools/rados/rados.cc

```diff
@@ -3342,18 +3342,25 @@ static int rados_tool_common(const std::map < std::string, std::string > &opts,
       return 1;
     }
 
-    set<string> out_keys;
-    ret = io_ctx.omap_get_keys(nargs[1], "", LONG_MAX, &out_keys);
-    if (ret < 0) {
-      cerr << "error getting omap key set " << pool_name << "/"
-	   << nargs[1] << ": "  << cpp_strerror(ret) << std::endl;
-      return 1;
-    }
+    string last_read = "";
+    int MAX_READ = 512;
```

Comment:
why 512?

Reply:
just want to keep consistent with the same role variable in “listomapvals”, it can be bigger

Reply:
could promote this magic number and the one used by `listomapvals` as a global `static const` variable, see also `default_op_size`.

Reply:
ok, have made it a global static const variable